pinot-common/src/main/java/com/linkedin/pinot/common/config/SegmentAssignmentStrategyConfig.java

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.common.config;
+
+import org.codehaus.jackson.annotate.JsonIgnore;
+import org.codehaus.jackson.annotate.JsonIgnoreProperties;
+
+/**
+ * Class representing configurations related to segment assignment strategy.
+ *
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SegmentAssignmentStrategyConfig {
+
+  private String _segmentAssignmentStrategy;
+  private String _numInstancesPerQueryPerPartition;
```

Comment:
Directly store int (Integer if need to use null to identify no config) instead of String. This way can use the json getter, no need for a separate getter.